.editorconfig

```diff
@@ -16,16 +18,12 @@ end_of_line = lf
 # Formatting - remove any whitespace characters preceding newline characters
 trim_trailing_whitespace = true
```

Comment:
`trim_trailing_whitespace = true` should apply to all the files.

Reply:
I tend to agree.